### PR TITLE
Pelagus naming and branding

### PR DESCRIPTION
**Pull request by 0xalank**

### CODE REVIEW

**Comment by wizeguyy** [comment]
please squash all these commits. They're effectively the same unit of work: rebrand taho->pelagus

**Comment by 0xalank** [comment]
> please squash all these commits. They're effectively the same unit of work: rebrand taho->pelagus

Done

File: background/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "@ledgerhq/hw-transport": "^6.20.0",
     "@ledgerhq/hw-transport-webusb": "^6.20.0",
     "@redux-devtools/remote": "^0.7.4",
-    "hd-keyring": "file:/Users/jonathan/hd-keyring",
+    "hd-keyring": "file:/Users/alan/code/hd-keyring",
```
Comment (gameofpointers) [inline background/package.json]: shouldn't we use a package here ?

File: background/redux-slices/migrations/to-29.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // This migration ensures the assets collection is cleared of duplicated
-// base assets https://github.com/tahowallet/extension/issues/3445
+// base assets https://github.com/pelaguswallet/pelagus/issues/3445
```
Comment (gameofpointers) [inline background/redux-slices/migrations/to-29.ts]: i think this should refer to the tahowallet issue

File: rfb/rfb-5-control-compatile-addresses.md [hunk 3]
```diff
@@ -131,7 +131,7 @@ another network:
     doesn't currently have an abstracted concept of this across hardware wallets,
     and most of the Ledger code is itself written with the assumption of using the
     Ethereum app (though [PR
-    2577](https://github.com/tahowallet/extension/pull/2577) has introduced some
+    2577](https://github.com/pelaguswallet/pelagus/pull/2577) has introduced some
```
Comment (gameofpointers) [inline rfb/rfb-5-control-compatile-addresses.md]: should link to tahowallet

File: ui/pages/Settings.tsx [hunk 4]
```diff
@@ -32,19 +32,19 @@ import SharedTooltip from "../components/Shared/SharedTooltip"
...
   },
   {
     icon: "twitter",
-    linkTo: "https://twitter.com/taho_xyz",
+    linkTo: "https://twitter.com/PelagusWallet",
   },
   {
     icon: "icons/m/github",
-    linkTo: "https://github.com/tallyhowallet/extension",
+    linkTo: "https://github.com/PelagusWallet/pelagus-extension",
```
Comment (gameofpointers) [inline ui/pages/Settings.tsx]: link is wrong, it should point to the pelagus and not pelagus extension

File: ui/pages/Settings.tsx [hunk 5]
```diff
@@ -32,19 +32,19 @@ import SharedTooltip from "../components/Shared/SharedTooltip"
 
 const NUMBER_OF_CLICKS_FOR_DEV_PANEL = 15
 const FAQ_URL =
-  "https://notion.taho.xyz/Tally-Ho-Knowledge-Base-4d95ed5439c64d6db3d3d27abf1fdae5"
+  "https://notion.pelaguswallet.io/Tally-Ho-Knowledge-Base-4d95ed5439c64d6db3d3d27abf1fdae5"
```
Comment (gameofpointers) [inline ui/pages/Settings.tsx]: do these links work? I see many places where we have replaced taho with pelaguswallet directly

File: yarn.lock [hunk 6]
```diff
@@ -8484,6 +8484,7 @@ hash.js@1.1.7, hash.js@^1.0.0, hash.js@^1.0.3, hash.js@^1.1.7:
     "@ethersproject/strings" "5.6.1"
     "@ethersproject/wallet" "5.6.1"
     bip39 "3.0.4"
+    quais "0.1.11"
```
Comment (gameofpointers) [inline yarn.lock]: does this quais version work for etxs? we dont want to break functionality for the things jonathan built

File: ui/components/Wallet/Banner/useBannerCampaigns.ts [hunk 7]
```diff
@@ -32,7 +32,7 @@ const bannerCampaigns: Campaign[] = [
       },
       secondary: {
         title: i18n.t("wallet.banner.learnMore"),
-        link: "https://blog.taho.xyz/dogyssey-launch/?utm_campaign=arbitrum-support&utm_medium=in-wallet&utm_source=dogyssey-banner",
+        link: "https://blog.pelaguswallet.io/dogyssey-launch/?utm_campaign=arbitrum-support&utm_medium=in-wallet&utm_source=dogyssey-banner",
```
Comment (gameofpointers) [inline ui/components/Wallet/Banner/useBannerCampaigns.ts]: this link doesnt seem to work

File: background/package.json (hunk 1 above)
Comment (jdowning100) [reply background/package.json]: We should use this https://github.com/PelagusWallet/pelagus-hd-keyring

File: .github/workflows/test-list/release-test-list.md [hunk 8]
```diff
@@ -196,16 +196,16 @@ This release checklist should be performed before release is published.
 ### 🎭 Sign in with Ethereum
 
 - [ ] SIWE on [login.xyz](https://login.xyz/)
-- [ ] SIWE on [Taho Pledge](https://taho.xyz/web3pledge)
+- [ ] SIWE on [Pelagus Pledge](https://pelaguswallet.io/web3pledge)
```
Comment (gameofpointers) [inline .github/workflows/test-list/release-test-list.md]: this link doesnt seem to work

File: yarn.lock (hunk 6 above)
Comment (jdowning100) [reply yarn.lock]: yarn.lock is autogenerated and it's using the same version that I used (in background/package.json)

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -13,26 +13,26 @@ to them very soon! More below.
 
 ## Deciding What to Work On
 
-Taho is currently being built by a core team in collaboration with a community
-on the [Taho Community Discord server](https://chat.taho.xyz). **Discord is
+Pelagus is currently being built by a core team in collaboration with a community
+on the [Pelagus Community Discord server](https://chat.pelaguswallet.io). **Discord is
```
Comment (jdowning100) [inline CONTRIBUTING.md]: Doesn't work either

File: background/package.json (hunk 1 above)
Comment (0xalank) [reply background/package.json]: Yeah we need to publish this

File: background/redux-slices/migrations/to-29.ts (hunk 2 above)
Comment (0xalank) [reply background/redux-slices/migrations/to-29.ts]: Right

File: rfb/rfb-5-control-compatile-addresses.md (hunk 3 above)
Comment (0xalank) [reply rfb/rfb-5-control-compatile-addresses.md]: right

**Comment by jdowning100** [comment]
I'm not sure what we want to do about links to features that we haven't implemented yet (on the website or otherwise). Do we want many broken links? Perhaps we should just leave them be?

File: .github/workflows/test-list/release-test-list.md (hunk 8 above)
Comment (0xalank) [reply .github/workflows/test-list/release-test-list.md]: We don't have a useful reference here. Will remove it.

File: CONTRIBUTING.md (hunk 9 above)
Comment (0xalank) [reply CONTRIBUTING.md]: Good catch

**Comment by 0xalank** [comment]
> I'm not sure what we want to do about links to features that we haven't implemented yet (on the website or otherwise). Do we want many broken links? Perhaps we should just leave them be?

We should probably just remove them.

**Review by wizeguyy — APPROVED** [review]

**Review by gameofpointers — APPROVED** [review]

**Pull request merged**